.github/workflows/c3-checks.yml

```diff
@@ -1,34 +0,0 @@
-name: C3 Checks
```

Comment:
Are you removing this and having the checks all run together? I didn't notice changes regarding that, was it already happening and we had this extra yml unnecessarily? 😮

Reply:
I renamed the C3 test job to `test:ci`, which runs it automatically

Reply:
isn't `test:ci` only running for wrangler and miniflare as Skye mentioned here: https://github.com/cloudflare/workers-sdk/pull/4325/files#r1384796683 ?

Reply:
It turns out the `vitest run` at the start of that command runs every test except miniflare & wrangler anyway

Reply:
ah ok thanks 🙂👍